cmd/ef_tests/levm/report.rs

```diff
@@ -353,6 +357,69 @@ impl fmt::Display for AccountUpdatesReport {
         for levm_updated_account_only in self.levm_updated_accounts_only.iter() {
             writeln!(f, "  {levm_updated_account_only:#x}:")?;
             writeln!(f, "{}", "    Was updated in LEVM but not in REVM".red())?;
+            let initial_account = self
+                .initial_accounts
+                .get(levm_updated_account_only)
+                .unwrap();
+            let updated_account_update = self
+                .levm_account_updates
+                .iter()
+                .find(|account_update| &account_update.address == levm_updated_account_only)
+                .unwrap();
+            let updated_account_storage = updated_account_update
+                .added_storage
+                .iter()
+                .map(|(key, value)| {
+                    let storage_slot = StorageSlot {
+                        original_value: *value,
+                        current_value: *value,
+                    };
+                    (*key, storage_slot)
+                })
+                .collect();
+            let updated_account_info = updated_account_update.info.clone().unwrap();
+            let updated_account = Account::new(
+                updated_account_info.balance,
+                updated_account_update.code.clone().unwrap_or_default(),
+                updated_account_info.nonce,
+                updated_account_storage,
+            );
+            if initial_account.info.balance != updated_account.info.balance {
+                writeln!(
+                    f,
+                    "{}",
+                    format!(
+                        "      Balance updated: {initial_balance} -> {updated_balance}",
+                        initial_balance = initial_account.info.balance,
+                        updated_balance = updated_account.info.balance
+                    )
+                    .red()
+                )?;
+            } else if initial_account.info.nonce != updated_account.info.nonce {
```

Comment:
Why do we use an `else if` here? Maybe we want to know if both balance and nonce are modified in the account

Reply:
You're right! I used `else if` to know if the updated account was added to that set incorrectly. I'll refactor this.

Reply:
Done in [10eecf7](https://github.com/lambdaclass/ethrex/pull/1280/commits/10eecf7c5ca31793bde0a247708ca647d3c9156c).